psalm.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
```

Comment:
Psalm came from original skeleton. But I have not solved all its complains, so psalm tests are not used.

Reply:
nevím zda to tu nechávat když to nepoužíváme?